### ISSUE TITLE
VSCode 1.28.0-insider doesn't refresh snippets when it's modified outside of VSCode

### ISSUE THREAD

**Issue by yangshuairocks** [Microsoft/vscode#59939]
- VSCode Version:
Version: 1.28.0-insider
Commit: 431ef9da3cf88a7e164f9d33bf62695e07c6c2a9
Date: 2018-10-03T12:44:29.343Z
Electron: 2.0.9
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64

- OS Version:
Win 10 Ultimate

Steps to Reproduce:

1. Open VSCode
2. Use another text editor software like notepad and create a snippet. Save it in the snippet folder. An example would be the file `latex.coordinate.code-snippets` with the following content:
```
{
    "Define a point": {
        "scope": "latex",
        "prefix": "coordinate",
        "body": [
            "testtt"
        ],
        "description": "Define a point"
    }
}
```
3. Now in VSCode, create a new latex tab and type `coordinate` to trigger the snippet. It will not work. You have to restart VSCode in order for it to pick up the newly created snippet. This is also true if you modified an existing snippet outside of VSCode. 

Note, this effect doesn't happen in the released version 1.27. 

Does this issue occur when all extensions are disabled?: Yes

**Comment by yangshuairocks** [comment]
Thanks man!

**Comment by yangshuairocks** [comment]
Is this fix included in Version: 1.28.0? I tested with it and the issue still exists. @jrieken

**Comment by jrieken** [comment]
no, this isn't in 1.28 but 1.29-insider
